run_tcpdump.sh

```diff
@@ -0,0 +1 @@
+nohup sudo tcpdump -i eth0 'port 28900' -v &> ./logs/tcpdump28900.log &
```

Comment:
Maybe add  comment so person who will be running it makes sure if "eth0"  interface exists? Because I saw on some AWS instance interface named differently: https://gyazo.com/3fa85fd6684e412679ea45befcbc2607

Reply:
Added